README.md

```diff
@@ -57,25 +57,25 @@ img.shape  # returns tuple of dimension sizes in TCZYX order
 img.get_image_data("CZYX", T=0)  # returns 4D CZYX numpy array
 
 # Get 5D TCZYX numpy array
-data = imread("my_file.tiff")
+data = imread("my_file.tiff")  # selects the first scene found
```

Comment:
If my friend told me "look at scene 37 in this file", then I can't really use imread unless we allow to pass a scene parameter here.

Reply:
I am somewhat okay with that personally but I can see an argument for adding the parameter as well.